.azure-pipelines/Linux-CI.yml

```diff
@@ -1,5 +1,6 @@
 trigger:
 - main
+- gh-readonly-queue/**
```

Comment:
Is it related to this PR?

Reply:
The code here is from main branch and so updating branch in the PR should hide these irrelevant changes..

Reply:
Can we resolve this, so it can be merged? Or, does anyone have a concern? (Seems to me it should happen automatically in git)